src/wren_compiler.c

```diff
@@ -894,7 +898,7 @@ static void ignoreNewlines(Compiler* compiler)
 // discards any duplicate newlines following it.
 static bool consumeLine(Compiler* compiler, const char* errorMessage)
 {
-  bool result = consume(compiler, TOKEN_LINE, errorMessage);
+  bool result = consume(compiler, TOKEN_LINE, errorMessage) != NULL;
```

Comment:
No, it wouldn't. The "consume(...)" function returns a pointer and this is only a special case check. (same applies for the other similiar change).